### ISSUE TITLE
Add dataset: [arabic_htr_groundtruth]

### ISSUE THREAD

**Issue by adikeinan** [bigscience-workshop/lam#76]

**Comment by davanstrien** [comment]
@adikeinan, thanks for suggesting thi!  I just realised the dataset suggestion template was broken (it should be fixed now). Is it this dataset: https://bl.iro.bl.uk/collections/ffab19f8-e9b7-485c-a7c3-64aba143be9c?locale=en. I'll open a new issue for this using the template if os.

**Comment by adikeinan** [comment]
Yes it is! It's well described there, but let me know if you need any additional details. Thanks!
